vocab/credentials/v2/vocab.csv

```diff
@@ -0,0 +1,23 @@
+JsonSchemaValidator2018,rdfs:Class, Json Schema Validator 2018,,,,,,A type of validator that can be used to syntactically validate JSON documents using the JSON Schema language.
+ManualRefreshService2018,rdfs:Class,Manual Refresh Service 2018,cred:RefreshService,,,,,A type of refresh service that must be interacted with in a manual fashion.
+RefreshService,rdfs:Class,Refresh Service,,,,,,A refresh service is a mechanism that can be utilized by software agents to retrieve an updated copy of a `verifiable credential`.
+VerifiableCredential,rdfs:Class,Verifiable Credential,,,,,,"A `credential` is a set of one or more claims made by an issuer. A `verifiable credential` is a tamper-evident credential that has authorship that can be cryptographically verified. Verifiable credentials can be used to build `verifiable presentations`, which can also be cryptographically verified. The claims in a credential can be about different subjects."
+VerifiableCredentialGraph,rdfs:Class,Verifiable Credential Graph,,,,,,"Instances of this class identify RDF Graphs, where each of these graphs must include exactly one Resource of type `VerifiableCredential`"
+VerifiablePresentation,rdfs:Class,Verifiable Presentation,,,,,,"A `presentation` is data derived from one or more `credentials`, issued by one or more `issuers`, that is shared with a specific `verifier`. A `verifiable presentation` is a tamper-evident `presentation` encoded in such a way that authorship of the data can be trusted after a process of cryptographic verification. Certain types of verifiable presentations might contain data that is synthesized from, but do not contain, the original verifiable credentials (for example, zero-knowledge proofs)."
+credentialSchema,rdf:Property,Credential Schema,,VerifiableCredential,cred:CredentialSchema,@id,,The value of the `credentialSchema` property MUST be one or more data schemas that provide verifiers with enough information to determine if the provided data conforms to the provided schema.
+credentialStatus,rdf:Property,Credential Status,,VerifiableCredential,cred:CredentialStatus,@id,,"The value of the `credentialStatus` property MUST include the `id` property, which MUST be a URL, and the `type` property, which expresses the credential status type (also referred to as the credential status scheme), which MUST provide enough information to determine the current status of the credential (for example, suspended or revoked)."
+credentialSubject,rdf:Property,Credential Subject,,VerifiableCredential,,@id,,An entity about which claims are made.
+evidence,rdf:Property,evidence,,VerifiableCredential,cred:CredentialEvidence,,,The value of the `evidence` property MUST be one or more evidence schemes providing enough information to a verifier to determine whether the evidence gathered meets their requirements for issuing a credential. The precise content of each evidence scheme is determined by the specific evidence type definition.
```

Comment:
The ranges of `credentialSchema`, `credentialStatus` and `evidence` are classes in the `cred:` namespace, but are not defined in the vocabulary.

Reply:
Oops.

@msporny I would need a one sentence description (label) to be added to the vocabulary for the following classes:

- cred:CredentialSchema
- cred:CredentialStatus
- cred:CredentialEvidence

For the time being I have added the class definition with the label set to T.B.D. but a proper sentence would be good.

Reply:
I make this resolved now, although the descriptions may require review.